src/BSON/DBPointer.c

```diff
@@ -103,19 +102,19 @@ static PHP_METHOD(DBPointer, __toString)
 */
 static PHP_METHOD(DBPointer, jsonSerialize)
 {
-	php_phongo_dbpointer_t *intern;
+	php_phongo_dbpointer_t* intern;
 #if PHP_VERSION_ID >= 70000
-	zval                    zdb_pointer, zoid;
+	zval zdb_pointer, zoid;
 #else
-	zval                   *zdb_pointer, *zoid;
+	zval *zdb_pointer, *zoid;
```

Comment:
Note: If you come up with a `ZVAL_RETVAL_TYPE` compat macro, you're going to need to break these into their own lines.